sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/First.scala

```diff
@@ -45,6 +45,17 @@ case class First(child: Expression, ignoreNullsExpr: Expression) extends Declara
 
   override def children: Seq[Expression] = child :: Nil
```

Comment:
Yea, this looks like a cleaner fix. Let me try whether it breaks anything else. Thanks!